release_notes/xpaas_release_notes.adoc

```diff
@@ -186,6 +186,10 @@ deploymentconfig "s2i-quickstart-karaf-camel-rest-sql" created
 +
 Workaround: Set SERVICE_NAME template parameter to *`karaf-camel-rest-sql*`.
 
+* https://issues.jboss.org/browse/OSFUSE-195[https://issues.jboss.org/browse/OSFUSE-195]: karaf-camel-amq template should have parameter to configure A-MQ service name
++
+The application template for AMQ deployments uses suffix for every transport type to distinguish. Hence there should be a configurable parameter for setting service name as envvironmental parameter *`A_MQ_SERVICE_NAME*`. 
```

Comment:
s/uses suffix/uses a suffix/

s/setting service name/setting the service name/

s/envvironmental parameter/environment variable/